spec/grape/dsl/configuration_spec.rb

```diff
@@ -70,6 +70,16 @@ class Dummy
           expect(subject.namespace_setting(:description)).to eq(expected_options)
           expect(subject.route_setting(:description)).to eq(expected_options)
         end
+
+        it 'can be set with options and a block' do
+          expect(subject).to receive(:warn)
```

Comment:
Add a match here to some text, just in case some other future `warn` is triggered.